packages/localize/test/number/formatNumber.test.js

```diff
@@ -259,19 +259,19 @@ describe('formatNumber', () => {
         expect(formatNumber(123456.789, currencyCode('USD'))).to.equal('123 456,79 USD');
         expect(formatNumber(123456.789, currencyCode('JPY'))).to.equal('123 457 JPY');
         expect(formatNumber(123456.789, currencySymbol('EUR'))).to.equal('123 456,79 €');
-        expect(formatNumber(123456.789, currencySymbol('USD'))).to.equal('123 456,79 $');
+        expect(formatNumber(123456.789, currencySymbol('USD'))).to.equal('123 456,79 $US');
         expect(formatNumber(123456.789, currencySymbol('JPY'))).to.equal('123 457 ¥');
```

Comment:
For `fr-FR` and `fr-BE` we did a forceYenSymbol, i dont know if it should be `¥`, `JP¥` (like other countries) or `¥JP` (just like the France do `$US`). Left it as is for now.